h/static/scripts/directive/form-input.coffee

```diff
@@ -0,0 +1,33 @@
+module.exports = ->
+  link: (scope, elem, attr, [model, validator]) ->
+    return unless  model
```

Comment:
Are you sure that nothing relies on the fact that this link function was previously a no-op if no validator was provided?